src/libs/actions/Report.js

```diff
@@ -285,11 +285,13 @@ function subscribeToReportCommentEvents() {
     if (Pusher.isSubscribed(pusherChannelName) || Pusher.isAlreadySubscribing(pusherChannelName)) {
         return;
     }
-
+    console.debug('Resubscribing to channel');
```

Comment:
We don't really need this log.

Reply:
Yeah. I will clean it and other logs added by me or the leftover comments of the old code.